.github/workflows/deploy-js-hello-world.yml

```diff
@@ -31,15 +31,12 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
-      - name: Setup Pages
-        uses: actions/configure-pages@v2
-      - name: Build with Jekyll
-        uses: actions/jekyll-build-pages@v1
-        with:
-          source: ./javascript-injectors/hello-world/pages
-          destination: ./_site/hello-world
-      - name: Upload artifact
-        uses: actions/upload-pages-artifact@v1
+      - name: Build Pages
+        working-directory: ./javascript-injectors/hello-world/pages
+        run: | 
+          chmod +x ./build.sh
```

Comment:
Feels a bit overkill with a script with just two lines. Just run it here instead as it makes it easier to debug.

Reply:
Made the change.